functions/index.js

```diff
@@ -1,33 +1,21 @@
 
 const functions = require('firebase-functions');
-var admin = require("firebase-admin");
-
-admin.initializeApp({
-    credential: admin.credential.applicationDefault(),
-    databaseURL: functions.config().audioseva.database_url,
-    storageBucket: functions.config().audioseva.storage_bucket
-});
-
-const bucket = admin.storage().bucket();
-const db = admin.database();
-
-//Functions
-const importMp3IntoSQR = require('./src/importMp3IntoSQR');
-const syncStorageToDB = require('./src/syncStorageToDB');
-
-// Helper Functions
-// 1. storeFileNameToDB( filePath, db_object )
-const helpers = require('./helpers');
-
-
-exports.importMP3IntoSQR = functions.storage.object().onFinalize( object => {
-    return importMp3IntoSQR.handleNewUploads(object, db, helpers.storeFileNameToDB);
-});
-
-
-exports.syncStorageToDB = functions.https.onRequest((req, res) => {
-    syncStorageToDB.handleCurrentlyUploadedFiles(bucket, db, helpers.storeFileNameToDB);
-    syncStorageToDB.removeNonExistingMp3DBEntries(bucket, db, helpers.removeFileNameFromDB);
-    return res.send(`Started Execution, the process is now Running in the background`);
+const admin = require("firebase-admin");
+
+try {
+    admin.initializeApp({
+        credential: admin.credential.applicationDefault(),
+        databaseURL: functions.config().audioseva.database_url,
+        storageBucket: functions.config().audioseva.storage_bucket
+    });
+} catch (err) { console.log(err) }
+
+//Loading the Functions from their modules
+const glob = require('glob');
+const functionFiles = glob.sync('./src/*.functions.js', { cwd: __dirname });
+
+functionFiles.forEach((file, index) => {
+    const filename = file.split('/').pop().slice(0, -13); // remove the trailing ".functions.js"
+    exports[filename] = require(file);
```

Comment:
Add comment here on what will the exported function names.

Reply:
I'm sorry I didn't understand what you mean here

Reply:
Add a block comment before this code and describe how the function name is constructed from the  file name and the JS function. You can give an example.